Source/WebCore/rendering/RenderLayerScrollableArea.cpp

```diff
@@ -2048,5 +2050,17 @@ void RenderLayerScrollableArea::invalidateScrollAnchoringElement()
         m_scrollAnchoringController->invalidateAnchorElement();
 }
 
+void RenderLayerScrollableArea::registerScrollbars()
+{
+    createScrollbarsController();
```

Comment:
We should make this safe to call more than once (i.e. add `ensureScrollbarsController()`).

Reply:
I added an explanation of the solution to the changelog but we need to change the ScrollbarsController type if we are going to keep the scrollbars in the web process, so this should remain a call to `createScrollbarsController`.